gum/backend-linux/gummemory-linux.c

```diff
@@ -87,6 +87,31 @@ gum_clear_cache (gpointer address,
 #endif
 }
 
+gboolean
```

Comment:
It would be great to move this function between `gum_memory_write` and `gum_clear_cache` to stay consistent with the order in the header file (gummemory.h).